docs/queues.txt

```diff
@@ -68,32 +82,41 @@ If you want to use MongoDB to handle failed jobs, change the database in
      - Description
 
    * - ``driver``
-     - **Required**. Specifies the queue driver to use. Must be ``mongodb``.
+     - **Required** Queue driver to use. The value of
+       this property must be ``mongodb``.
 
    * - ``connection``
-     - The database connection used to store jobs. It must be a ``mongodb`` connection. The driver uses the default connection if a connection is not specified.
+     - Database connection used to store jobs. It must be
+       a ``mongodb`` connection. The driver uses the default connection
+       if a connection is not specified.
 
    * - ``collection``
-     - Name of the MongoDB collection to store failed jobs. Defaults to ``failed_jobs``. 
-
+     - Name of the MongoDB collection to store failed
+       jobs. The value is ``failed_jobs`` by default.
 
-Add the service provider in ``config/app.php``:
+Then, add the service provider in your application's
+``config/app.php`` file:
 
 .. code-block:: php
 
    MongoDB\Laravel\MongoDBQueueServiceProvider::class,
 
-
 Job Batching
 ------------
 
-`Job batching <https://laravel.com/docs/{+laravel-docs-version+}/queues#job-batching>`__
-is a Laravel feature to execute a batch of jobs and subsequent actions before,
-after, and during the execution of the jobs from the queue.
+**Job batching** is a Laravel feature that enables you to execute a
+batch of jobs and subsequent actions before, after, and during the
+execution of the jobs from the queue. To learn more about this feature,
```

Comment:
S: I think the use of "subsequent" is a little misleading, since these actions can also run before or during the execution of the jobs
```suggestion
batch of jobs and related actions before, after, and during the
execution of the jobs from the queue. To learn more about this feature,
```